.env.demo

```
@@ -392,8 +395,8 @@ SECRETS_PROVIDER=openbao
 BAO_URL=http://127.0.0.1:8200
 # Default KV v2 path fetched at boot and injected into process.env
 BAO_SECRET_PATH=secret/data/credebl_resend_api_key
-# AppRole credentials. Generate your own via: ./openbao-init.sh
-# (values below are for the local 'openbao' container created by docker-compose.openbao.yml)
-BAO_ROLE_ID=ca2a350b-1871-0851-8557-42b36aa9c623
-BAO_SECRET_ID=3f0d8d45-c232-f5c1-2984-fe58e5144be8
+# AppRole credentials. Provision your own outside version control via: ./openbao-init.sh
+# The values below are non-functional placeholders; never commit real role/secret IDs.
+BAO_ROLE_ID=00000000-0000-0000-0000-000000000000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BAO_ROLE_ID key should go before the BAO_SECRET_PATH key

(UnorderedKey)